### PR TITLE
add Dockerfile.

### PR DESCRIPTION
**Pull request by cdoron**
based on:
https://github.com/fybrik/fybrik/blob/master/connectors/katalog/Dockerfile

Signed-off-by: Doron Chen <cdoron@il.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,14 @@ GIT_REPO_ID_CLIENT := datacatalog-go-client
 
 FYBRIK_VERSION ?= v1.0.1
 
+DOCKER_HOSTNAME ?= ghcr.io
+DOCKER_NAMESPACE ?= fybrik
+DOCKER_TAG ?= 0.0.0
+DOCKER_NAME ?= openmetadata-connector
+
+IMG := ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/${DOCKER_NAME}:${DOCKER_TAG}
+export HELM_EXPERIMENTAL_OCI=1
```
Comment (roytman) [inline Makefile:18]: we don't need this line

File: Makefile [hunk 2]
```diff
@@ -62,3 +70,15 @@ generate-code:
 patch: generate-code
 	awk '($$1 != "\"github.com/gorilla/mux\"") {print}' auto-generated/api/go/api_default.go > ${TMP_FILE}
 	mv ${TMP_FILE} auto-generated/api/go/api_default.go
+
+.PHONY: build
```
Comment (roytman) [inline Makefile:74]: this rule is called `source-build` in fybrik, can you rename it pls.

File: Makefile [hunk 3]
```diff
@@ -62,3 +70,15 @@ generate-code:
...
+build: compile
+	docker build . -t ${IMG}; cd ..
+
+.PHONY: docker-push
+docker-push:
```
Comment (roytman) [inline Makefile:79]: can you change it according to https://github.com/fybrik/fybrik/blob/d97245e50c1d0384046a9b601d0166c821617f38/hack/make-rules/docker.mk#L11

File: Makefile (hunk 2 above)
Comment (cdoron) [reply Makefile:74]: replaced `compile` with `source-build`
and `build` with `docker-build`

**Review by roytman — APPROVED** [review]

**Comment by cdoron** [comment]
addresses https://github.com/fybrik/openmetadata-connector/issues/43

**Pull request merged**